core/domain/collection_services.py

```diff
@@ -1122,7 +1122,13 @@ def get_next_page_of_all_commits(
 
     Returns:
         3-tuple of (results, cursor, more) as described in fetch_page() at:
-        https://developers.google.com/appengine/docs/python/ndb/queryclass
+        https://developers.google.com/appengine/docs/python/ndb/queryclass,
+        where:
+            results: list(CollectionCommitLogEntry). List of query results.
```

Comment:
Here and below, just to check: is this a CollectionCommitLogEntry (domain object) or a CollectionCommitLogEntryModel (ndb storage model)? They are different things...